tox.ini

```
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{38,39,310}-dj{32,41}-wagtail{41,42}
-    py{310,311}-dj{41}-wagtail{41}
-    py{310,311}-dj{42}-wagtail{50,main}
+    py{38,39,310}-dj{32}-wagtail{41,42,50,51}
+    py{38,39,310,311}-dj{41}-wagtail{41,42}
+    py{310,311}-dj{42}-wagtail{50,51,main}
```

While I think the above is OK, I think we should standardize the envlist across these sites.
For instance, the above doesn't have tests for py{x]-dj41-wagtail{50,51}

```suggestion
    py{38,39,310}-dj{32,41}-wagtail{41,42,50,51}
    py311-dj41-wagtail{41,42,50,51}
    py311-dj42-wagtail{50,51}
```

---

Hi @katdom13 Thank you this is a good idea.